### PR TITLE
For updated image

### PR DESCRIPTION
**Pull request by thenaomig**
Merge this so that climakitae works with the updated image on the hub. Makes some small changes to account for new versions of a few packages.

### CODE REVIEW

**Comment by elehmer** [comment]
I added `intake-esm` to requirements.txt and environment.yml as it is now needed.

**Comment by thenaomig** [comment]
Need to update requirements.txt and environment.yml to match the version numbers here: https://github.com/pangeo-data/pangeo-docker-images/blob/2023.02.08/pangeo-notebook/conda-linux-64.lock

**Comment by thenaomig** [comment]
Someone will need to double-check my version numbers because now tests are failing...

**Comment by elehmer** [comment]
> Someone will need to double-check my version numbers because now tests are failing...

On it...

File: requirements.txt [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-scipy==1.7.1
-shapely==1.7.1
+xarray==2023.2.0
+cartopy==0.21.1
+dask-gateway==2023.1.1
+geopandas==0.12.2
+geoviews==1.9.6
+holoviews==1.15.4
+hvplot==0.8.2
+intake==0.4.0
```
Comment (elehmer) [inline requirements.txt:8]: `intake==0.6.6`

File: requirements.txt [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+intake-xarray==0.6.1
+intake-esm==2022.9.18
```
Comment (elehmer) [inline requirements.txt:10]: Add `intake-geopandas==0.4.0`?

File: environment.yml [hunk 3]
```diff
@@ -4,28 +4,32 @@ channels:
...
-  - shapely=1.7.1
-  - xarray=0.19.0
+  - cartopy=0.21.1
+  - dask-gateway=2023.1.1
+  - dask-geopandas
+  - geopandas=0.12.2
+  - geoviews=1.9.6
+  - holoviews=1.15.4
+  - hvplot=0.8.2
+  - intake=0.4.0
```
Comment (elehmer) [inline environment.yml:14]: `intake==0.6.6`

**Comment by thenaomig** [comment]
Thanks for finding my intake version error, @elehmer. I'm currently tempted to merge a version of this PR without the pinned version-numbers so that something working is on the hub. And then open a separate PR to pin the versions, after we figure out why this test is failing. Thoughts?

**Comment by elehmer** [comment]
> Thanks for finding my intake version error, @elehmer. I'm currently tempted to merge a version of this PR without the pinned version-numbers so that something working is on the hub. And then open a separate PR to pin the versions, after we figure out why this test is failing. Thoughts?

It is worth a shot. I tried bumping python to 3.10.9 like it is on the HUB but just caused another failure. We are in pyproj/proj hell again. I am not sure how it got built on the HUB?

File: Makefile [hunk 4]
```diff
@@ -1,23 +1,7 @@
-PKGNAME = climakitae
```
Comment (bkg) [inline Makefile:1]: Looks like you overwrote the `Makefile` by accident, could you roll this back please?

Comment (thenaomig) [reply Makefile]: Whoops. Fixed.

**Pull request merged**